docs/dev-guide/api-ref.md

```diff
@@ -392,12 +392,60 @@ The response code will be `201` is successful.
 
 #### Fetch a specific mediator
 
-`GET /mediators/:uuid`
+`GET /mediators/:urn`
 
-where `:uuid` is the `uuid` property of the mediator to fetch.
+where `:urn` is the `urn` property of the mediator to fetch.
 
 The response status code will be `200` if successful and the response body will contain a mediator object. See the [mediator schema](https://github.com/jembi/openhim-core-js/blob/master/src/model/mediators.coffee).
 
+#### Mediator heartbeat endpoint
+
+This endpoint allows a mediator to send a heartbeat to the OpenHIM-core. This serves two purposes:
+
+1. It allows the mediator to demonstrate is alive-ness and submit an uptime property
+2. It allow the mediator to fetch the latest configuration from the OpenHIM-core
+
+This endpoint only returns mediator config if the config has changed between the time the previous heartbeat was recieved and the current time. You may force the endpoint to return the latest config via the `config: true` property.
+
+`POST /mediators/:urn/heartbeat`
+
+where `:urn` is the `urn` property of the mediator that is sending in a heartbeat.
+
+with an http body of:
+
+```javascript
+{
+  "uptime": 50.25 // The uptime is seconds
+  "lastHeartbeat": '2015-09-17T13:41:07.784Z' // A date in ISO 8601 format
```

Comment:
Would it not be possible for the HIM to just use `new Date()` for the last heartbeat [here](https://github.com/jembi/openhim-core-js/blob/190e9a89ee2762226f23f8692c3235bb98e1c353/src/api/mediators.coffee#L126)

Then the mediators don't need to send this property through? It would always be now() anyway right? Or do you see a need for mediators to send a past datetime? (.. what if they send a future datetime?)